packaging/linux/smoketest/smoketest_all.bash

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/bash
+set -euox pipefail
+echo "Starting linux smoketests"
+
+versionstring=$1
+version=$(echo "$versionstring" | cut -d'-' -f1)
+datetime=$(echo "$versionstring" | cut -d'-' -f2 | cut -d'+' -f1)
+revision=$(echo "$versionstring" | cut -d'+' -f2)
+curl --output /dev/null --silent --head --fail "https://s3.amazonaws.com/jack-testing.keybase.io/linux_binaries/deb/keybase_${version}-${datetime}.${revision}_amd64.deb"
+curl --output /dev/null --silent --head --fail "https://s3.amazonaws.com/jack-testing.keybase.io/linux_binaries/rpm/keybase-${version}.${datetime}.${revision}-1.x86_64.rpm"
+for platform in */; do
+    case $platform in
+        vagrantcommon/) continue;;
+    esac
+    echo "testing $platform"
+    (
+        cd "$platform"
+        vagrant snapshot restore --no-start default
+        vagrant up
+        vagrant rsync
+        set +e
+        ssh -o "UserKnownHostsFile /dev/null" -o "StrictHostKeyChecking no" \
```

Comment:
lol these options

Reply:
Its actually the same ones vagrant uses, see `vagrant ssh-config`.